flamedisx/x1t_sr1.py

```diff
@@ -2,11 +2,15 @@
 """
 import numpy as np
 import tensorflow as tf
+import tensorflow_probability as tfp
 
 from multihist import Hist1d
 import wimprates
 
 import flamedisx as fd
+import os 
```

Comment:
Have a look at the import order: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Shifted to top.